INSTALL_DOCKER.rst

```diff
@@ -153,41 +120,14 @@ Then run the following commands to add those group ids to your active login sess
     newgrp kvm
     newgrp docker
 
-Download the cc-oci-runtime 2.0 code
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. attention::
-   Update this section when cc-oci-runtime is available in a Clear Linux bundle
-
-Download, build and install the ``cc-oci-runtime`` from source:
-
-  ::
-
-    cor_source=${HOME}/cc-oci-runtime
-    git clone https://github.com/01org/cc-oci-runtime.git $cor_source
-    cd $cor_source
-    bash autogen.sh --disable-cppcheck --disable-valgrind
-    make
-    sudo make install
-
-Restart Docker
-~~~~~~~~~~~~~~
-
-In order to ensure you are running the latest installed Docker_ you should restart the Docker_ daemon:
-
-  ::
-
-    sudo systemctl daemon-reload
-    sudo systemctl restart docker-upstream
-
 Final Docker sanity check
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Before we dive into using `Clear Containers`_ it is prudent to do a final sanity check to ensure that relevant Docker_ parts have installed and are executing correctly:
 
   ::
 
-    sudo systemctl status docker-upstream
+    sudo systemctl status docker
     docker-upstream ps
```

Comment:
Should be `docker` now, not `docker-upstream`. What does `systemctl restart docker` give you?